README.md

```diff
@@ -56,6 +56,17 @@ Maintainers can trigger the [release workflow](.github/workflows/release.yaml) b
 
 We wanted to backfill `release-0.1` to `release-0.12` to have a changelog, but we were not able to use a GitHub action in a newer commit to trigger a release that generates a changelog on older commits. See #489 for full discussion.
 
+## Metrics Deprecation
+
+The following recording rule is marked deprecated. They will be removed in v2.0.0.
+```bash
+node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate
+```
+It will be replaced by the following recording rule to preserve data points using `rate` and add `5m` to indicate rate in the recording rule name.
```

Comment:
```suggestion
It will be replaced by the following recording rule to preserve data points using `rate` and add `5m` to indicate the range of the rate query in the recording rule name.
```